sources/packages/workflow/src/deploy.models.ts

```diff
@@ -0,0 +1,26 @@
+export const PROCESSES_EXTENSION = ".bpmn";
+export const DECISIONS_EXTENSION = ".dmn";
+export const DEPLOYMENT_METADATA_PROPERTY_NAME = "Metadata";
+
+export enum DeploymentMetadataTypes {
+  DecisionRequirements = "decisionRequirements",
+  Decision = "decision",
+}
+
+export interface DecisionDeploymentResult {
+  requirementsId: string;
+  requirementsName: string;
+  requirementsKey: number;
+  resourceName: string;
```

Comment:
resouseName can be null too right.  `resourceName: string | null;` 
![image](https://user-images.githubusercontent.com/77353155/202534158-28bc7185-d720-4449-8815-8e139d1777d0.png)